src/config/accounts.c

```diff
@@ -952,6 +950,17 @@ accounts_get_login_presence(const char* const account_name)
     return result;
 }
 
+char*
+accounts_get_login_status(const char* const account_name)
+{
+    gchar* setting = g_key_file_get_string(accounts, account_name, "presence.login", NULL);
+    gchar* status = NULL;
+    if (strcmp(setting, "last") == 0) {
```

Comment:
maybe use g_strcmp0() in case `setting` is NULL?

Reply:
I used normal `strcmp` because that was done [here](https://github.com/profanity-im/profanity/blob/22cda3dbea49d8ac9382117427ec4889bf999ff8/src/config/accounts.c#L939), but `g_strcmp0` would probably be better.